packages/sdks/src/constants/builder-registered-components.ts

```diff
@@ -31,5 +33,6 @@ export const getDefaultRegisteredComponents: () => RegisteredComponent[] =
     { component: Slot, ...slotComponentInfo },
     { component: Symbol, ...symbolComponentInfo },
     { component: Text, ...textComponentInfo },
+    { component: Tabs, ...tabsComponentInfo },
```

Comment:
so we do not need to exclude this when `TARGET === 'rsc'` after all?

Reply:
yup! although it still doesn't work because its a client component rendering `<Blocks>`, but no need to exclude as its not breaking or interfering with other tests - what do you think?

Reply:
Let's exclude it. Otherwise customers will see it in their visual editor and try to use it despite it being broken.

Reply:
Makes sense! Will update it